softvis3d-frontend/src/services/sonarqube/SonarQubeComponentInfoService.ts

```diff
@@ -0,0 +1,64 @@
+///
+/// softvis3d-frontend
+/// Copyright (C) 2016 Stefan Rinderle and Yvo Niedrich
+/// stefan@rinderle.info / yvo.niedrich@gmail.com
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU Lesser General Public
+/// License as published by the Free Software Foundation; either
+/// version 3 of the License, or (at your option) any later version.
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+/// Lesser General Public License for more details.
+///
+/// You should have received a copy of the GNU Lesser General Public
+/// License along with this program; if not, write to the Free Software
+/// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+///
+import { BackendService } from "./BackendService";
+import { AppStatusStore } from "../../stores/AppStatusStore";
+import ErrorAction from "../../classes/status/ErrorAction";
+
+export interface SonarQubeComponentInfo {
+    id: number;
+    key: string;
+    name: string;
+    analysisDate: Date;
+}
+
+export default class SonarQubeComponentInfoService extends BackendService {
+    private static LOAD_COMPONENT_INFO_ERROR_KEY: string = "LOAD_COMPONENT_INFO_ERROR";
+
+    private appStatusStore: AppStatusStore;
+    private projectKey: string;
+
+    constructor(projectKey: string, appStatusStore: AppStatusStore, baseUrl?: string) {
+        super(baseUrl);
+
+        this.projectKey = projectKey;
+        this.appStatusStore = appStatusStore;
+    }
+
+    public loadComponentInfo(): Promise<SonarQubeComponentInfo> {
+        return new Promise<SonarQubeComponentInfo>((resolve, reject) => {
+            const params = {component: this.projectKey};
```

Comment:
> ## GET api/components/show
> [...] The 'id' or 'key' must be provided.

This should be `{key: this.projectKey}`. At least with SonarQube Version 6.3.1.

Reply:
This one is funny. They changed the request parameter of the api/components/show webservice with SQ 6.4. Description in 

6.3.1: "The 'id' or 'key' must be provided."
6.4: "The 'componentId' or 'component' parameter must be provided."

I took the time to check with which revision / issue this change happened and it was commited with this revision: 
https://github.com/SonarSource/sonarqube/commit/a914082d1e284d97f6d1d632ab04e8ba4eff30f7

Unfortunately, there is no more SONAR-8774 as it says in the commit message:
https://jira.sonarsource.com/browse/SONAR-8774

And i could not find any mentioning of that in the release noted or change logs. Sometimes its really funny how they just change stuff.

To make it simple, i changed the implementation that it does not break 6.3.1. As the parameter analysisDate is also only available from 6.4, the auto reload feature is now disabled in 6.3.1.